.github/fix-docs-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# This script takes care of setting proper version in docs.
+#
+set -eo pipefail
+
+VERSION=$1
+
+update_file_versions() {
+  local VERSION="$1"
+  local FILE="$2"
+  sed -i "s/<version>.*<\/version>/<version>${VERSION}-lib<\/version>/g" README.md
```

Comment:
consider changing  `README.md` with `${FILE}` so we can use that function for other files replacement.